packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -242,6 +242,7 @@ export interface IOpsCachingPolicy {
      * At the same time, big number means we wait for so many ops to accumulate, which
      * increases chances and number of trailing ops that would not be flushed to cache
      * when user closes tab
+     * Use any number below 1 to disable caching
```

Comment:
would be good to add this case in packages\test\test-drivers\src\odspDriverApi.ts:

```
export const odspOpsCaching: OptionsMatrix<IOpsCachingPolicy> = {
    batchSize: [undefined, -1],
    timerGranularity: numberCases,
    totalOpsToCache: numberCases,
};
```

Reply:
done